src/nix/path-info.cc

```diff
@@ -45,6 +46,13 @@ struct CmdPathInfo : StorePathsCommand, MixJSON
             .description = "Show signatures.",
             .handler = {&showSigs, true},
         });
+
+        addFlag({
+            .longName = "--query-substitutes",
+            .shortName = 'U',
```

Comment:
Not sure whether we should introduce a `shortName` for this. Those should only be used for flags that will be used a lot.

Reply:
Should be fine. I will mostly be using this in scripts anyway, where the long name makes the code easier to follow.